recipes/libvorbis/CMakeLists.txt

```diff
@@ -0,0 +1,108 @@
+cmake_minimum_required (VERSION 3.1.0)
+project (VORBIS)
```

Comment:
The xiph organisation are working on their own CMakeLists.txt (see https://github.com/xiph/vorbis/blob/master/CMakeLists.txt), but they haven't included it with any of their releases yet, so I guess it's still WIP. For future versions of this package, I will use the CMakeLists.txt by the package, but for this version I will stick with what works if that's ok.